### ISSUE TITLE
Philo's PvP Series: Concussive Shot is Dispellable

### ISSUE THREAD

**Issue by PhilosophyOfZen** [elysium-project/classic-bug-tracker#1066]
**Summary:** 
Hunter's Concussive Shot is dispellable by Priest's Dispel Magic.

**Steps to Reproduce:** 
Have a hunter shoot a priest with Concussive Shot. Dispel it.

**Expected Results:** 
Concussive shot should be a physical effect. Paladin BoP should dispel it for instance, or limited invulnerability potion, but not dispel magic.

**Actual Result:**
Concussive shot is dispelled.

**Additional Information:**

**Comment by lehurleur** [comment]
Wrong. The spell for the charge is physical, but the spell for the stun aura is magic.
Sources: [https://classicdb.ch/?spell=13327](url), [https://classic.wowhead.com/spell=13327/reckless-charge](url)
